.github/workflows/e2e-k8s.yml

```diff
@@ -61,12 +69,20 @@ jobs:
           -Dmaven.checkstyle.skip \
           -Dmaven.deploy.skip \
           -Ddocker.push.skip=true \
-          -Pdocker,release -Ddocker.tag=ci \
+          -Prelease -Ddocker.tag=ci \
           -pl org.apache.dolphinscheduler:dolphinscheduler-alert-server \
           -pl dolphinscheduler-tools \
           -pl dolphinscheduler-api \
           -pl dolphinscheduler-master \
           -pl dolphinscheduler-worker -am
+      - name: Build Image
+        run: |
+          components=("master" "worker" "api" "tools" "alert-server")
+          for component in "${components[@]}"; do
+            docker build --build-arg JAVA_VERSION=${{ matrix.java }} \
+            -t apache/dolphinscheduler-${component}:ci \
+            -f dolphinscheduler-${component}/src/main/docker/Dockerfile
+          done
```

Comment:
Please take a look this   [plugin](https://github.com/apache/dolphinscheduler/blob/ce11674668cf26747f34c3474763c32e1ba5b6d1/pom.xml#L455-L517).

You can add a new environment variable for the Java version in the plugin and pass it to the Docker CLI.